README.md

```diff
@@ -97,49 +93,53 @@ For example, given the following HTML Content:
 
 ```html
 <columns>
-	<column>
-		<p>Example paragraph in Column<p>
-	</column>
-<column>
+  <column>
+    <p>Example paragraph in Column</p>
+    <p></p
+  ></column>
+
+  <column></column
+></columns>
 ```
 
 It will return the following blocks:
 
 ```json
 [
-	{
-	  "__typename": "CoreColumns",
-		"name": "core/columns",
-		"id": "63dbec9abcf9d",
-		"parentClientId": null
-	},
-	{
-	  "__typename": "CoreColumn",
-	  "name": "core/column",
-	  "id": "63dbec9abcfa6",
-      "parentClientId": "63dbec9abcf9d"
-	},
-	{
-	  "__typename": "CoreParagraph",
-	  "name": "core/paragraph",
-	  "id": "63dbec9abcfa9",
-      "parentClientId": "63dbec9abcfa6",
-	  "attributes": {
-	    "content": "Example paragraph in Column 1",
-	    "className": null
-	  }
-	}
+  {
+    "__typename": "CoreColumns",
+    "name": "core/columns",
+    "id": "63dbec9abcf9d",
+    "parentClientId": null
+  },
+  {
+    "__typename": "CoreColumn",
+    "name": "core/column",
+    "id": "63dbec9abcfa6",
+    "parentClientId": "63dbec9abcf9d"
+  },
+  {
+    "__typename": "CoreParagraph",
+    "name": "core/paragraph",
+    "id": "63dbec9abcfa9",
+    "parentClientId": "63dbec9abcfa6",
+    "attributes": {
+      "content": "Example paragraph in Column 1",
+      "className": null
+    }
+  }
 ]
 ```
 
 The `CoreColumns` contains one or more `CoreColumn` block, and each `CoreColumn` contains a `CoreParagraph`.
 
-Given the flattened list of blocks though, how can you put it back? Well that's where you use the `` and `parentId` fields to assign temporary unique ids for each block.
+Given the flattened list of blocks though, how can you put it back? Well that's where you use the ``and`parentId` fields to assign temporary unique ids for each block.
```

Comment:
``and`parentId` should be and `parentClientId`. I had the impression that we have updated the README for the new changes but apparently not.

Reply:
Good catch! Fixed.